site/pages/institution-landing-en.hbs

```diff
@@ -0,0 +1,188 @@
+			</div>
+		</div>
+	</div>
+</div>
+
+<section class="well well-sm provisional gc-most-requested">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-2">
+				<h2>Most requested</h2>
+			</div>
+			<div class="col-md-10">
+				<ul class="colcount-md-2">
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+				</ul>
+			</div>
+		</div>
+	</div>
+</section>
+
+<div class="container mrgn-bttm-lg">
+	<section class="gc-srvinfo">
+		<h2 class="wb-inv">Services and information</h2>
+		<div class="wb-eqht row">
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+		</div>
+	</section>
+</div>
+
+<div class="well well-sm brdr-0 mrgn-bttm-0">
+	<div class="container">
+		<section class="provisional gc-about">
```

Comment:
@duboisp @delisma GC About gives the bold on lists for what is considered a ".gc-about" section.

Reply:
@jmealing is the bold used to reduce the focus on the minister’s section? Should we think about a different approach if that's the case, because it might have an inverse effect? When everything is bold and a few elements unbold, I feel like the unbold elements attract more focus.

Reply:
@delisma No, it's bold to carry the same visual weight as the doormat links above, and the news article titles below. We used bold during testing for the CRA contact us project (where they specifically had to use the contact us section). We saw no issue. We could bold the Minister names to match the visual weight as well for those links

Reply:
@GormFrank This seems to be a type of page layout, like a "well-bold" kind of. We will need to ensure the sub-elements like "strong" need to visually show emphasis. You can also write a set of rules (that is programmable) to specify when to use and when to not use (Integration testing).